common/subdevice-model.cpp

```diff
@@ -992,7 +992,7 @@ namespace rs2
         };
 
         auto col0 = ImGui::GetCursorPosX();
-        auto col1 = 155.f;
+        auto col1 = 9.f * (float)config_file::instance().get( configurations::window::font_size );
```

Comment:
Why do we sometimes take the get_font_size() and sometimes the configuration?

Reply:
In the _viewer.cpp_ we have a window object so I call `window.get_font_size()`
In other places, I haven't a window object and I get it from an instance of config.

Reply:
OK